curriculum/js0/5.js

```diff
@@ -7,8 +7,16 @@
  */
 
 const solution = (a, b, c) => {
-  return 0;
-};
+  if (a>b && a>c) {
+    return a;
+  };
+  if (b>a && b>c) {
+    return b;
+  }
+  if (c>a && c>b) {
+    return c;
+  }
+   };
```

Comment:
next time you can simply rewrite it like this:
  
if (a>b && a>c) {
    return a;
  };
  if (b>a && b>c) {
    return b;
  }
return c;